fe/fe-core/src/test/java/com/starrocks/sql/plan/PlanTestBase.java

```diff
@@ -1293,7 +1293,7 @@ public void runFileUnitTest(String filename, boolean debug) {
     }
 
     public void runFileUnitTest(String filename) {
-        runFileUnitTest(filename, true);
```

Comment:
Why change this？

Reply:
This bug is introduced by this PR https://github.com/StarRocks/starrocks/commit/cb7c2814418938e64c70097c1eadcbd3096de5ac